oak_containers/proto/interfaces.proto

```diff
@@ -71,6 +71,6 @@ service Orchestrator {
   rpc GetApplicationConfig(google.protobuf.Empty) returns (GetApplicationConfigResponse) {}
 
   // Notifies the orchestrator that the trusted app is ready to serve requests and listening on the
-  // specified port.
+  // pre-arranged port (8080).
```

Comment:
Thanks! 

Just checking: the port the trusted app listens on changed recently from 8088 to 8080. 

Now the guest listens for everything from loading the stage1 until the app on the same port, which is defined here:

https://github.com/project-oak/oak/blob/cec1442b1b1d0755137d88912438d95a35a78533/oak_containers_launcher/src/qemu.rs#L144

Reply:
> Just checking: the port the trusted app listens on changed recently from 8088 to 8080.

The trusted app was always listening on port 8080 inside the VM. Port 8088 was a hardcoded port on the host that QEMU was proxying to the trusted app. #4197 changed the QEMU configuration so that 8088 is no longer hardcoded. It now uses a dynamically assigned port. This remapping of the host port to guest port is configured using `hostfwd`: https://github.com/project-oak/oak/blob/main/oak_containers_launcher/src/qemu.rs#L146

>Now the guest listens for everything from loading the stage1 until the app on the same port:

At the moment the trusted app is the only thing listening on a port in the guest. Stage1 and the orchestrator initiates connection rather than listening. The port happens to be the same, but the IP address is different, so the endpoint is different. Those connections from stage1 and the orchestrator to the host are proxied to the host via the `guestfwd` configuration you linked above.

Reply:
Thank you, @conradgrobler for the detailed answer, yes to all you say.

> The port happens to be the same

Maybe it makes sense to choose a different default port for one of those if opportunity arises.

And I think it'd help me to find a naming scheme for all the ports involved. I'll think about it, when I am back!